docs/errors.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Errors
+page_title: Errors | Kendo UI Internationalization Package
+description: "List of possible exceptions that the package could throw."
+slug: errors_internalization_kendoui
+position: 6
+---
+
+# Errors list
+
+Below is the list of all controlled errors that could be thrown:
+
+| Type                         | Message
+|:---                          |:---
+|`NoLocaleError`               | Missing locale info for {locale}
```

Comment:
The documentation section is manually written. It is not generated from the `src/error-details.js` list.